addb2/dump.c

```diff
@@ -655,7 +655,7 @@ static void sym(struct m0_addb2__context *ctx, const uint64_t *v, char *buf)
 		if (json_output)
 			sprintf(buf, "\"symbol\":{\"addr\":%"PRId64","
 				     "\"addr_ptr_wrap\":%"PRId64"}",
-				     (long)addr, v[0]);
+				(uint64_t)addr, v[0]);
```

Comment:
%PRIu64 required for uint64_t, same required at other places

Reply:
fixed in 915ed2b